app/src/pages/api/internal/v1/openapi.json.ts

```diff
@@ -0,0 +1,16 @@
+import { type NextApiRequest, type NextApiResponse } from "next";
+import { generateOpenApiDocument } from "trpc-openapi";
+import { v1ApiRouter } from "~/server/api/internal/v1Api.router";
+
+export const openApiDocument = generateOpenApiDocument(v1ApiRouter, {
+  title: "OpenPipe Docker API",
```

Comment:
"OpenPipe Internal API". Docker isn't a useful description. We may or may not use Docker as the runtime, and anyway, our main app runs in Docker too, so it's a weird thing to use as a distinguisher.